gradio/components/audio.py

```diff
@@ -110,6 +110,7 @@ def __init__(
         max_length: int | None = None,
         waveform_options: WaveformOptions | dict | None = None,
         loop: bool = False,
+        recording: bool = False,
```

Comment:
Add docstring

Reply:
I kinda expected this to start a Gradio app that was recording from the moment the page loads:

```py
import gradio as gr

with gr.Blocks() as demo:
    gr.Audio(sources=["microphone"], recording=True)

demo.launch()
```

Reply:
Or in this case, the start/stop buttons to stop recording:

```py
import gradio as gr

with gr.Blocks() as demo:
    start = gr.Button("Start")
    stop = gr.Button("Stop")
    a = gr.Audio(sources=["microphone"])
    start.click(lambda : gr.Audio(recording=True))
    stop.click(lambda : gr.Audio(recording=False))

demo.launch()
```

Reply:
Ok that was an oversight on my part. Should be fixed now but note that in the second demo you need to set the audio to be interactive explicitly (or pass it as input to your click events). You also need to set output events in the click definition.

Reply:
Beautiful!